docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-detect.md

```diff
@@ -0,0 +1,111 @@
+# Dasharo Compatibility: USB Detection
+
+## Test cases
+
+### USB001.001 USB detection after coldboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device after
+coldboot (reboot realized by power supply cutting off then cutting on). This 
+test case may be re-done several times to specify the platform and connection
+stability.
+
+**Test configuration data**
+
+1. `BOOT_MENU_KEY` = `ESC`
+2. `BOOT_MENU_STRING` = `Press ESC for boot menu`
+3. `USB_STICK` = `USB MSC Drive General USB Flash Disk 1100`
+
+**Test setup**
+
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power off the DUT.
```

Comment:
We should replace in the Test steps section the following statements: Power On/Off to Power cutting On/Off.